.github/workflows/workflow.yml

```diff
@@ -34,12 +41,21 @@ jobs:
       run: docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin <<< "${{ secrets.DOCKER_API_KEY }}"
     - name: Push image
       run: |
-        if [[ "${{ github.ref }}" =~ ^refs/tags/v(.*) ]] ; then
-          VERSION="${BASH_REMATCH[1]}"
-        elif [[ "${{ github.ref }}" =~ ^refs/heads/master ]] ; then
-          VERSION="latest"
-        else
-          VERSION="unknown"
-        fi
-
+        VERSION="latest"
         make publish-all VERSION=${VERSION}
+  publish-tag:
+    runs-on: ubuntu-latest
+    needs: [ lint, test ]
+    if: ${{ success() && startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Rather than using a tag of `vX.Y.Z/blah`, maybe we should use `blah/vX.Y.Z` so that it matches the way that Kpt [handles tags](https://googlecontainertools.github.io/kpt/guides/consumer/update/#fetch-output)?

Reply:
Yep, good idea